.github/workflows/workflow.yml

```
@@ -165,7 +169,8 @@ jobs:
         run: |
           TAG=${GITHUB_REF_NAME}
           docker pull navitia/loki:$TAG
-          docker tag navitia/loki:$TAG $PRD_ECR_REGISTRY/navitia-loki-loki:$TAG
+          docker build -t navitia/loki:aws-$TAG -f docker/loki_opentelemetry_dockerfile --build-arg LOKI_TAG=$TAG .
+          docker tag navitia/loki:aws-$TAG $PRD_ECR_REGISTRY/navitia-loki-loki:$TAG
```

Is it really specific AWS, or only OpenTelemetry. How about `otel`  instead?

Also, suffix is more used than prefix when there is a version: `debian:11-5-slim`, `rabbitmq:3.11.4-alpine`, etc.

```suggestion
          docker tag navitia/loki:$TAG-otel $PRD_ECR_REGISTRY/navitia-loki-loki:$TAG
```

---

I'll skip setting this temporary tag